docs/knowledge_base/threat_feed.md

```diff
@@ -157,13 +161,124 @@ Response:
     "Triaged Malware Rule": true
   },
   "malware": true,
+  "name": "adrmdr",
   "riskScores": {
     "author": 1.0,
     "engineering": 1.0,
     "license": 1.0,
     "malicious_code": 0.01,
     "total": 0.05,
     "vulnerability": 1.0
-  }
+  },
+  "version": "0.1.0"
```

Comment:
Maybe a nitpick, but maybe the response can match same order specified?

- `name`: The name of the package.
 - `version`: The version of the package.
 - `ecosystem`: The ecosystem to which the package was published.
 - `code`: The HTTP status code.
 - `malware`: A boolean indicating if the package has been triaged as malware.
 - `riskScores`: An object containing risk scores for each of the 5 Phylum risk domains associated with the pa